tools/check_release_build.py

```diff
@@ -29,6 +30,8 @@ def test_release_build(
         assert manifest_file.is_file()
         manifest = json.loads(manifest_file.read_text(encoding="utf-8"))
         assert "manifest_version" in manifest
+        command = shlex.split(manifest["command"])[0]
+        assert (dist_dir / command).exists()
```

Comment:
`manifest["command"]`もcommandなので意図がわかりづらそう。
こうかな。
```suggestion
        command_filename = shlex.split(manifest["command"])[0]
        assert (dist_dir / command_filename).exists()
```